db/dcrpg/upgrades.go

```diff
@@ -65,23 +68,23 @@ type VinVoutTypeUpdateData struct {
 	TxType     stake.TxType
 }
 
-// VotingMilestones defines the various milestones phases have taken place when
+// votingMilestones defines the various milestones phases have taken place when
 // the agendas have been up for voting. Only sdiffalgorithm, lnsupport and
 // lnfeatures agenda ids exist since appropriate voting mechanisms on the dcrd
 // level are not yet fully implemented.
-var VotingMilestones = map[string]dbtypes.MileStone{
+var votingMilestones = map[string]dbtypes.MileStone{
```

Comment:
This should work if we just declared `var votingMilestones = map[string]dbtypes.MileStone{}`.  I'm not a fan of hard coding data that we can and do request from the authoritative source.